.github/workflows/_reusable_build_package.yml

```diff
@@ -3,12 +3,6 @@ name: '_reusable_build_package'
 on:
   workflow_call:
     inputs:
-      otc_version:
-        required: true
-        type: string
-      otc_sumo_version:
-        required: true
-        type: string
       otc_build_number:
         required: false
```

Comment:
should this be required true? what about workflow_id?

Reply:
Yes, good catch.